.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = exporters/tests/*
```

Comment:
had to create this to exclude tests, that are inside exporters folder. I would create an issue to refactor and move tests to outside it, and remove this file